tests/model_serving/model_runtime/vllm/negative/test_error_proxying.py

```
@@ -0,0 +1,95 @@
+class TestErrorProxying:
+    def test_malformed_chat_request(
+        self,
+        vllm_inference_service: InferenceService,
+    ) -> None:
+        """Given a running vLLM ISVC,
+        When a malformed chat completion request is sent (messages as string instead of list),
+        Then the server returns HTTP 400/422 with an error message.
+        """
+        url = get_exposed_isvc_url(isvc=vllm_inference_service)
+        query = {**MALFORMED_CHAT_QUERY, "model": vllm_inference_service.name}
+        response = requests.post(f"{url}/v1/chat/completions", json=query, verify=False, timeout=30)
```

Ruff (0.15.18)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)